### PR TITLE
Test-1

### PR DESCRIPTION
**Pull request by dariaEvich**

### CODE REVIEW

File: src/pages/Inventory.page.js [hunk 1]
```diff
@@ -7,9 +7,41 @@ export class InventoryPage extends BaseSwagLabPage {
...
+
     addItemToCartButton = this.page.locator('[id^="add-to-cart"]');
 
+    filterDropdown = this.page.getByTestId('product-sort-container');
+
     async addItemToCartById(id) {
         await this.addItemToCartButton.nth(id).click();
     }
+
+    async sortFromLowToHighPrice() {
```
Comment (EugenePetrik) [inline src/pages/Inventory.page.js]: It might be better to create one method and switch/case inside the method

File: src/pages/Inventory.page.js [hunk 2]
```diff
@@ -7,9 +7,41 @@ export class InventoryPage extends BaseSwagLabPage {
...
+    }
+
+    async getInventoryItemNames() {
+        const names = await this.inventoryItemNames.allTextContents();
+        return names;
+    }
+
+    async getInventoryItemPrices() {
+        const prices = await this.inventoryItemPrices.allTextContents();
+        return prices;
```
Comment (EugenePetrik) [inline src/pages/Inventory.page.js:49]: Lets return a list of numbers without the `$` sign and update this method `sortFomLowToHigh`

**Review by EugenePetrik — CHANGES_REQUESTED** [review]
Very good job! Left a few comments

File: src/pages/Inventory.page.js (hunk 1 above)
Comment (dariaEvich) [reply src/pages/Inventory.page.js]: changed

File: src/pages/Inventory.page.js (hunk 2 above)
Comment (dariaEvich) [reply src/pages/Inventory.page.js:49]: Updated

**Review by EugenePetrik — APPROVED** [review]
Good job!

**Pull request merged**